packages/cardano-services/test/StakePool/StakePoolHttpService.test.ts

```diff
@@ -1359,10 +1362,14 @@ describe('StakePoolHttpService', () => {
       });
 
       it('rewardsHistoryLimit is correctly honored', async () => {
-        const noLimitResponse = await provider.queryStakePools({ pagination });
-        const limitOneResponse = await provider.queryStakePools({ pagination, rewardsHistoryLimit: 1 });
+        const limitThreeResponse = await provider.queryStakePools({ pagination, rewardsHistoryLimit: 3 });
+        const limitFiveResponse = await provider.queryStakePools({ pagination, rewardsHistoryLimit: 5 });
+        expect(limitThreeResponse.pageResults[0].metrics.apy).not.toBe(limitFiveResponse.pageResults[0].metrics.apy);
```

Comment:
If `apy` computation is based on this then maybe `0` is not a good default?

Reply:
I think you are right, I just inspected that the `apy` computation requires at least 3 epoch rewards, but with `rewardsHistoryLimit: 0` they are 0, an empty list.
So then maybe defaulting to 3 is more appropriate, given the fact `metrics.apy` is changing depending on each epoch reward from the history. @mkazlauskas

Reply:
Please like if you agree on 3 as the default value for the rewards history limit.